Resources/config/routing/sonata_change_password_2.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+
+<routes xmlns="http://symfony.com/schema/routing"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://symfony.com/schema/routing http://symfony.com/schema/routing/routing-1.0.xsd">
+
+    <route id="fos_user_change_password" pattern="/change-password">
```

Comment:
```pattern``` is [deprecated](https://github.com/symfony/Routing/blob/v2.2.0/CHANGELOG.md#220) from Symfony 2.2. You should use ```path``` instead.